style.css

```diff
@@ -19,10 +19,71 @@ body{
 	padding: 0;
 }
 
+/*hide scrollbar*/
 ::-webkit-scrollbar {
     display: none;
 }
 
+/*style for preloader starts*/
+.loading-area{
+	width: 100vw;
+	height: 100vh;
+	background-image: linear-gradient(#1155CC, #527ECC);
+	position: fixed;
+	left: 0;
+	top: 0;
+	z-index: 1000;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+.loading-area.done{
+	animation: loadingDone 3s;
+	animation-fill-mode: forwards;
+}
+
+@keyframes loadingDone {
+  0% {
+    opacity: 1;
+  }
+  70% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 0; 
+	display: none;
+	z-index: -1000;
+  }
+}
+/*style for loader*/
+.zoom-in-out-box {
+  width: 80px;
+  height: 80px;
+  animation: zoom-in-zoom-out 2s ease infinite;
+}
+
+@keyframes zoom-in-zoom-out {
+  0% {
+    transform: scale(1, 1);
+  }
+  50% {
+    transform: scale(1.5, 1.5);
+  }
+  100% {
+    transform: scale(1, 1);
+  }
+}
+
+.logo{
```

Comment:
#added space between the edge of the navbar and hover effects to expand the graphic